pkg/model/clusterstatus.go

```diff
@@ -10,6 +10,7 @@ const (
 	ClusterStatusDeletePending           Status = "delete_pending"
 	ClusterStatusDeleting                Status = "deleting"
 	ClusterStatusDeleteError             Status = "delete_error"
+	ClusterStatusDeleteErrorRetryable    Status = "delete_error_retryable"
```

Comment:
Please consider #528 - update of external API required before merging. We have to check with @piotrmiskiewicz how KEB is dealing with the new added statuses.